src/dsp_tools/utils/excel_to_json_lists.py

```diff
@@ -24,7 +24,7 @@
 
 def expand_lists_from_excel(
     lists_section: list[dict[str, Union[str, dict[str, Any]]]]
-) -> Tuple[list[dict[str, Any]], bool]:
+) -> list[dict[str, Any]]:
```

Comment:
having a success status is most important in the methods called by dsp_tools.py. Here, in the deeper levels, it's enough that the methods runs through without errors, and then it's clear that it ran through successfully.